server/controllers/authController/authController.test.js

```diff
@@ -222,7 +222,7 @@ describe('AuthController', () => {
       let appDb
 
       beforeAll(async () => {
-        appDb = await global.MONGO_INSTANCE.db()
+        appDb = await global.MONGO_INSTANCE.db('auth')
```

Comment:
what does specifying `'auth'` here do?

Reply:
it creates a db called auth, this is to stop collisions from happening by the other tests using the same resource.